extensions/wikia/AdEngine/js/context/slotsContext.js

```diff
@@ -27,7 +27,6 @@ define('ext.wikia.adEngine.context.slotsContext', [
 		var context = adContext.getContext(),
 			isHome = context.targeting.pageType === 'home',
 			isOasis = context.targeting.skin === 'oasis',
-			isPremiumAdLayoutEnabled = context.opts.premiumAdLayoutEnabled,
 			isIncontentEnabled =
 				!isHome &&
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[project-id]: wikia:app